pylint_plugin.py

```diff
@@ -0,0 +1,43 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+# We need this plugin because of two open pylint' issues:
+#   * https://github.com/PyCQA/pylint/issues/3882
+#   * https://github.com/PyCQA/pylint/issues/3876
+# Once these issues will be fixed we can update pylint and remove this plugin.
+
+from astroid import MANAGER, FunctionDef, inference_tip, extract_node
```

Comment:
from my experience  `astroid` tend to break too often, I would rather wait for a fixed version. (or dive in, and fix it at the source of it)
the only reason I would consider pylint plugins, is for writing our own checks, but I don't have any idea of what we could check, we should consider it regardless of moving to python3.9